packages/web/components/bridge/review-screen.tsx

```diff
@@ -227,9 +228,11 @@ const AssetBox: FunctionComponent<{
               height={16}
             />
           )}
-          <div className="text-wosmongton-200">
-            {shorten(address, { prefixLength: 12 })}
-          </div>
+          <Tooltip maxWidth="500px" content={address}>
```

Comment:
especially since the cursor is pointer, what if we allow the user to copy the address when they click? Then the tooltip could be momentarily changed to "copy". 

If you don't want to bother, then lets remove the cursor pointer style